### PR TITLE
update docs for reward distribution

### PR DESCRIPTION
**Pull request by jlevison**
Docs update as discussed, @ronnno please review

### CODE REVIEW

File: rewards-distribution/ethereum/client/README.md [hunk 1]
```diff
@@ -1,11 +1,27 @@
...
+* Run npm install, have truffle config up to date and relevant to your system
+* Ensure that the `truffle-config` is configured and that all environment variables are valid (ethereum url, mnemonic and so on)
+* **Check the gas prices and adjust it in the truffle config**
+
+## Creating the batches
+To begin working, in a terminal, navigate to the root of truffle project under `rewards-distribution/ethereum`
+You will need to then execute the batch generation script:
 ```$bash
-truffle exec client/getBatchHahses.js  [csv filename] [number of payments per batch]
+./node_modules/.bin/truffle exec client/getBatchHahses.js  [csv filename] [number of payments per batch] --network ropsten/mainnet/development
```
Comment (ronnno) [inline rewards-distribution/ethereum/client/README.md]: consider adding npm commands so that the instructions will read:
`npm truffle exec...`

Comment (ronnno) [reply rewards-distribution/ethereum/client/README.md]: consider replacing `ropsten/mainnet/development` with `[ropsten|mainnet|development]`

**Comment by jlevison** [comment]
@ronnno can you check why its failing, obviously not related to the changes made here, the ci is broken and its impossible to merge this :/

**Comment by ronnno** [comment]
problem in e2e seems related to transition to go modules. @itamararjuan does this look like the plugin issue? do we have a fix for this or still working on it?

**Comment by ronnno** [comment]
update - plugin issue is now resolved but this issue persists. @itamararjuan is investigating

**Pull request merged**